sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -305,6 +305,30 @@ def apply_minp(arr, ddof, minp):
     gen_hpat_pandas_series_rolling_ddof_impl(arr_var))
 
 
+@sdc_register_jitable
+def pop_count(value, nfinite, result):
+    """Calculate the window sum without old value."""
+    if not numpy.isnan(value):
+        result -= 1
+
+    return nfinite, result
+
+
+@sdc_register_jitable
+def put_count(value, nfinite, result):
+    """Calculate the window sum with new value."""
+    if not numpy.isnan(value):
+        result += 1
+
+    return nfinite, result
+
+
+@sdc_register_jitable
+def result_count(nfinite, minp, result):
```

Comment:
Maybe `count_result` as for method `mean`?

Reply:
No, here is different.

Reply:
Let it be.

Reply:
Let's generalize the function and name it just `result` or `get_result` without connections to the method `count`. Don't forget to change docstring. BTW I see mention about min periods in the docstring, but don't see usage of `minp` in the function.